src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,54 @@
+package duke.parser;
+
+
+import duke.command.*;
```

Comment:
Please avoid using wildcard imports. Consider listing out the classes you will be using instead.